java/ql/src/Advisory/Documentation/ImpossibleJavadocThrows.ql

```diff
@@ -11,22 +11,18 @@
 
 import java
 
-RefType getTaggedType(ThrowsTag tag) {
+Class getTaggedType(ThrowsTag tag) {
```

Comment:
Why Class not ClassOrInterface?

Reply:
Good point. Previously this also detected (unintentionally?) when an interface was erroneously documented as thrown type. I have changed this to `ClassOrInterface` now.